content/realtime/messages.textile

```diff
@@ -420,6 +421,126 @@ bc[swift]. channel.attach { error in
+
+  * Emoji reactions
+  * Message updating and deleting
+  * Read receipts and unread counts
+  * Quoting/replying to a message
+  * Message threads
+  * Media previews 
+
+  Message Interactions enable you to build such features by adding support for a new @ref@ object that allows you to indicate what the interaction represents, and which message it is referring to.
+
+  The @ref@ object contains two fields:
+
+  * @type@ (String) - a constant representing the reason for the interaction.
+  * @timeSerial@ (String) - the timeSerial of the message being referenced.
+
+  h3(#enabling-message-interaction). Enabling Message Interactions 
+
+  Before using message interactions, they must be  enabled for channels in the Ably dashboard:
+
+  1. Login to your "Ably dashboard":https://ably.com/login.
+  2. Select the app you are working with.
+  3. Click *Settings* on the right side of the navigation bar.
+  4. Scroll down to *Channel rules* and click *Add new rule*.
+  5. Give your new rule a Namespace or provide a specific Channel ID.
+  6. Select the *Message references enabled* checkbox then click *Create channel rule*.
+
+  <a href="/images/realtime/dmessage-interactions.png" target="_blank">
+  <img src="/images/realtime/message-interactions.png" style="max-width:80%" alt="Message interactions in dashboard">
+  </a>
+
+  You now have a namespace based rule you can apply to channels to enable Message Interactions or a specific channel with Message Interactions enabled.
+
+  To read more on applying rules to channels read our "channel rules documentation":https://ably.com/docs/general/channel-rules-namespaces/.
+
+  h3(#using-message-interaction). Using Message Interactions
+
+  Once Message Interactions have been enabled for a channel you can start to reference the @timeSerial@ of previous messages when you publish interactions. Messages sent to Message Interaction enabled channels will automatically include a @timeSerial@.
+
+  To reference a previous message, include the @ref@ object inside the @extras@ object including: 
+
+  * A @type@ constant string defining the reason for interaction.
+  * The @timeserial@ string of the interacted message.
+
+  *Note*: for @type@ constants Ably has reserved strings beginning: @com.ably.@.
+
+  For example, if you wanted to send an emoji reaction to the previous message above you could send something like:
+
+  bc[jsall]. const sendReaction = (emoji) => {
+      channel.publish('', { body: 😀, extras: { reference: { type: com.ably.chat.reaction, timeserial: "1656424960320-1" } } })
+  }
+
+  h3(#interactions-listener). Creating a Listener
+
+  Regular listeners will get all messages (including those containing a reference), but it is possible to filter specifically for messages with or without a reference by supplying a filter object. A @filter object@ can contain any number of the following fields:
+
+  * @refTimeserial@ (String) - filter containing a specific message timeserial (for example: ‘v1b25XrTDg:0’).
```

Comment:
> ‘v1b25XrTDg:0’

Odd combination of ticks. Perhaps use `@` instead? Same for items on the next line.